src/main/java/com/nylas/JobStatus.java

```diff
@@ -54,8 +61,16 @@ public String getJobObjectId() {
 
 	@Override
 	public String toString() {
-		return "JobStatus [action=" + action + ", createdAt=" + created_at + ", jobObjectId=" + getJobObjectId()
-				+ ", object=" + object + ", status=" + status+ ", accountId=" + getAccountId() + ", id=" + getId() + "]";
+		return "JobStatus [" +
```

Comment:
You can use `ReflectionToStringBuilder` to avoid typing all the fields: https://commons.apache.org/proper/commons-lang//apidocs/org/apache/commons/lang3/builder/ReflectionToStringBuilder.html

Reply:
@quzhi1 I really need to switch the toStrings to something like this... thanks for the reccomendation!